src/content/structured/components/data-table/accessibility.mdx

```diff
@@ -0,0 +1,81 @@
+---
+path: "/components/data-table/accessibility"
+
+date: "2025-05-14"
+
+title: "Data table"
+
+status: "CANARY"
+
+subTitle: "Use a data table to organise and display large quantities of related data."
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/data-table" },
+    { title: "Code", path: "/components/data-table/code" },
+    { title: "Accessibility", path: "/components/data-table/accessibility" },
+  ]
+
+tags: ["Table"]
+---
+
+## Easy to use for everyone
+
+Data tables present information in a structured format, with interactive elements such as sorting and pagination controls designed to be keyboard accessible. Users can navigate through the main controls without unnecessary tab stops, making interaction more efficient.
+
+Cell content can be truncated using `truncation-pattern` (`truncationPattern`) and will either be displayed as a [tooltip](/components/tooltip) or a “See more/see less” toggle. Tooltips will appear on hover. When a “See more/see less” pattern is used, the toggle is fully keyboard accessible and included in the tab order, allowing users to expand or collapse content easily.
+
+Sorting controls within the data table column headers are implemented as a [button](/components/button) and can be focused and activated via keyboard using the ‘Enter’ or ‘Space’ keys.
+
+Any slotted action elements in cells are also added into the data table’s tabbing order and are fully keyboard navigable.
+
+The data table’s title bar contains interactive elements such as a density [select](/components/select) and slotted action elements that are all reachable using keyboard navigation. Focus styling is applied consistently to ensure users can visually track where they are within the title bar.
+
+When the data table’s pagination bar is used, [pagination](/components/pagination) buttons follow a logical tab order, with disabled pagination buttons being skipped, preventing interaction with inactive controls.
+
+## For Assistive Technology
+
+The data table uses semantic HTML elements such as `<table>`, `<thead>`, `<tbody>`, `<th>`, and `<td>`, which aids screen readers to correctly interpret and announce the table structure and relationships between headers and cells.
+
+Sorting controls use a polite ARIA live region, giving real-time updates about which column is sorted and in what order.
+
+An assertive ARIA live region is also used to notify screen reader users about dynamic changes in the loading/updating state.
+
+## Based on
+
+<ul>
+  <li>
+    <ic-link
+      target="_blank"
+      href="https://www.w3.org/WAI/ARIA/apg/patterns/table/"
+      rel="noreferrer noopener nofollow"
+    >
+      Web Content Accessibility Guidelines (WCAG) 2.2, World Wide Web Consortium
```

Comment:
Not entirely clear what both these links are for as they display the same text. Some more descriptive titles would be useful